schematic/store/synapse.py

```diff
@@ -762,7 +762,7 @@ def uplodad_manifest_file(self, manifest, metadataManifestPath, datasetId, restr
         # update file name
         file_name_full = metadataManifestPath.split('/')[-1]
         file_extension = file_name_full.split('.')[-1]
-        file_name_new = 'synapse_storage_manifest_' + component_name + '.' + file_extension
+        file_name_new = CONFIG["synapse"]["manifest_basename"] + component_name + '.' + file_extension
```

Comment:
Can we add update the reading from the CONFIG to `os.path.basename(CONFIG["synapse"]["manifest_basename"])`? I believe in some cases users could want to store the manifest(s) in a directory other than their working directory and this would allow for that without errors while also matching previous style: #647 (and before). I see now though that my parameter naming in the config could have been more clear to avoid this confusion, apologies